documentation/learnmore/existing.md

```diff
@@ -150,12 +148,11 @@ Successfully validated 1 migration (execution time 00:00.111s)
 Current version of schema "public": 1
 Schema "public" is up to date. No migration necessary.</pre>
 
-We can now see that our production database is already at version `1` which matches the `-baselineVersion` we specified during the `baseline` command. This means it is up to date with our migration scripts and nothing needs to be run.
+We can see that our Production database is already at version `1` which matches the `-baselineVersion` we specified to the `baseline` command. This means it is up to date with our migration scripts and nothing needs to be run.
 
 ### Next steps
 
-If you have followed through this tutorial using our example and are ready to integrate Flyway into your own database, don't forget that [Spawn](https://spawn.cc) is available for free to create fast isolated copies for development and testing of your own databases. Enabling you to perfect the baselining process on a production like copy before the real thing. Find out how to create a data image of your own database [here](https://spawn.cc/docs/source-configuration-backup-postgres).
-
+If you have followed our examples and are ready to integrate Flyway into your own database, don't forget that [Spawn](https://spawn.cc) is available for free to create fast, isolated copies for development and testing of your own databases. Enabling you to perfect the baselining process on a copy of Production before touching the real database. You can find out how to create a data image of your own database [here](https://spawn.cc/docs/source-configuration-backup-postgres).
```

Comment:
The sentence beginning with `Enabling` reads a little odd, and would flow better if it was `your own databases, enabling`

Reply:
I think it becomes a very long sentence then, how about `This enables you`?

Reply:
Perfect!

Reply:
Done